mne/epochs.py

```diff
@@ -1374,6 +1374,13 @@ def save(self, fname, split_size='2GB', fmt='single', verbose=True):
             will slightly differ due to the reduction in precision.
 
             .. versionadded:: 0.17
+        overwrite : bool
+            If True, the destination file (if it exists) will be overwritten.
+            If False (default), an error will be raised if the file exists.
+            To overwrite original file (the same one that was loaded),
+            data must be preloaded upon reading.
```

Comment:
... This defaults to True in 0.18 but will change to False in 0.19.

Reply:
Done, thanks!